components/Navbar.vue

```diff
@@ -240,7 +234,7 @@ const account = computed(() => identityStore.getAuthAddress)
 const isCreateVisible = computed(() => createVisible(urlPrefix.value))
 
 const logoSrc = computed(() => {
-  const variant = isMobile.value ? 'Koda' : 'Koda_Beta'
+  const variant = isTouch ? 'Koda' : 'Koda_Beta'
```

Comment:
If you want to address #8767, this line isn't needed. You can wrap HTML something like this

```vue
<nuxt-link to="/" class="navbar-item logo nuxt-link-active">
  <!-- logo on mobile -->
  <img
    src="/Koda.svg"
    alt="First NFT market explorer on Kusama and Polkadot"
    class="lg:hidden" />
	<!-- logo on desktop -->
  <img
    src="/Koda_Beta.svg"
    alt="First NFT market explorer on Kusama and Polkadot"
    class="hidden lg:block" />
</nuxt-link>
```

Reply:
you just need to wrap it for dark mode